tests/integration/cli/tests/run.rs

```diff
@@ -21,6 +21,66 @@ fn test_no_start_wat_path() -> String {
     format!("{}/{}", ASSET_PATH, "no_start.wat")
 }
 
+#[cfg(any(target_os = "linux", target_os = "macos"))]
+#[test]
+fn test_cross_compile_python_windows() -> anyhow::Result<()> {
+    let temp_dir = tempfile::TempDir::new()?;
+
+    let targets = &[
+        "aarch64-darwin",
+        "x86_64-darwin",
+        "x86_64-linux-gnu",
+        "aarch64-linux-gnu",
+        // TODO: this test depends on the latest release -gnu64.tar.gz
+        // to be present, but we can't release the next release before
+        // the integration tests are passing, so this test depends on itself
+        //
+        // We need to first release a version without -windows being tested
+        // then do a second PR to test that Windows works.
+        // "x86_64-windows-gnu",
```

Comment:
Is it worth adding a link to the ticket so we don't forget about this?

Reply:
No because I know I need to fix this before 3.0.0, so the ticket would only be up for a few hours.